libs/SalesforceSDK/res/layout/sf__login.xml

```diff
@@ -1,13 +1,29 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:id="@+id/sf__auth_container_phone"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:fillViewport="true">
+    android:layout_height="match_parent">
 
-    <WebView android:id="@+id/sf__oauth_webview"
-        android:layout_width="match_parent"
+    <ScrollView android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:scrollbars="none" />
-</ScrollView>
\ No newline at end of file
+        android:fillViewport="true"
+        android:layout_above="@+id/sf__idp_login_button">
+
+        <WebView android:id="@+id/sf__oauth_webview"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:scrollbars="none" />
+    </ScrollView>
+
+    <Button android:id="@+id/sf__idp_login_button"
```

Comment:
Reorganized the layout to add a `Launch IDP app` button at the bottom of the login screen to trigger IDP flow on an enabled app. This button is shown only if the IDP login flag is enabled.